aws-synthetics-group/aws-synthetics-group.json

```diff
@@ -0,0 +1,115 @@
+{
+    "typeName": "AWS::Synthetics::Group",
+    "description": "Resource Type definition for AWS::Synthetics::Group",
+    "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-synthetics",
+    "properties": {
+        "Name": {
+            "description": "Name of the group.",
+            "type": "string",
+            "pattern": "^[0-9a-z_\\-]{1,64}$"
+        },
+        "Id": {
+            "description": "Id of the group",
+            "type": "string"
+        },
+        "Tags" : {
+            "type" : "array",
+            "uniqueItems" : false,
+            "items" : {
+                "$ref" : "#/definitions/Tag"
+            },
+            "minItems": 0
+        },
+        "ResourceArns": {
+            "type" : "array",
+            "uniqueItems" : true,
+            "items" : {
+                "$ref" : "#/definitions/ResourceArn"
+            },
+            "maxItems": 20
+        }
+    },
+    "definitions" : {
+        "Tag": {
+            "description": "A key-value pair to associate with a resource.",
+            "additionalProperties" : false,
+            "type": "object",
+            "properties": {
+                "Key": {
+                    "type": "string",
+                    "description": "The key name of the tag. You can specify a value that is 1 to 127 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -. ",
+                    "minLength": 1,
+                    "maxLength": 128,
+                    "pattern": "^(?!aws:)([a-zA-Z\\d\\s_.:/=+\\-@]+)$"
+                },
+                "Value": {
+                    "type": "string",
+                    "description": "The value for the tag. You can specify a value that is 1 to 255 Unicode characters in length and cannot be prefixed with aws:. You can use any of the following characters: the set of Unicode letters, digits, whitespace, _, ., /, =, +, and -. ",
```

Comment:
Does it accept Unicode chars? Regex suggests otherwise.

Reply:
Copy the descriptions from this https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html

Reply:
I have used the canary tag descriptions since we use the same API behind scenes